### DISCUSSION TITLE
如何刷入非stock版本的大分区AX3600固件

### DISCUSSION THREAD

**Discussion by zlopen** [immortalwrt/immortalwrt#1015] [Q&A]
### Describe the bug

当前在AX3600上用的immortalwrt stock版本固件，想了解基于immortalwrt的场景下，如何刷入非stock版本的大分区immortalwrt固件
看[openwrt官方的说法](https://openwrt.org/toh/xiaomi/ax3600#install_the_firmware)是先刷入initramfs-factory.ubi，然后重启后再刷sysupgrade.bin，不知道基于immortalwrt我该怎么办呢？
看到网上有说uboot兼容问题也会引发问题，事关分区，所以我还挺怕失误的，求指导

### ImmortalWrt version

r23381-3810066f16

### ImmortalWrt target/subtarget

ipq807x/generic

### Device

Xiaomi AX3600 (stock layout)

### Image kind

Self-built image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
必须恢复原厂分区和 uboot 才能刷。不推荐。

**Comment by douglarek** [comment]
你的意思你现在有 uboot 了不满意，想刷个不 uboot 的？挺有想法的，用小米官方工具恢复原厂系统再按 openwrt 步骤来就行（刷对应的后缀包）。

**Reply by 1715173329** [reply]
刷了第三方 uboot 之后恢复工具就没用了
